db/schema.rb

```diff
@@ -265,7 +265,7 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "dynamic_annotation_fields", id: :serial, force: :cascade do |t|
+  create_table "dynamic_annotation_fields", id: :integer, default: -> { "nextval('new_dynamic_annotation_fields_id_seq'::regclass)" }, force: :cascade do |t|
```

Comment:
@caiosba I just rebased my branch off of develop and am running into an issue on this line when I try to use `bin/rails db:test:prepare`. I had missed it during review because I had looked at the migrations but not schema changes. do you know why this line changed, or what `nextval('new_dynamic_annotation_fields_id_seq'::regclass)` references?

Reply:
@hartsick I don't know! I didn't change it manually... I wonder if it was generated by some older version I was running. Feel free to overwrite it with a newer version.

Reply:
gotcha! I wonder if it's an old change hanging around in your local database? I now see that function is referenced in `[doc/structure.sql](https://github.com/meedan/check-api/tree/d0dc4cab212c62194c66d64d171d577b1293412d/doc/structure.sql)`. I'll see if I can change it back

Reply:
ah yes, probably old data I have here...

Reply:
it looks like some indexes below also got removed on this table - annotation_id, annotation, and value. that was also unintentional, right? just checking before I re-add

Reply:
> it looks like some indexes below also got removed on this table - annotation_id, annotation, and value. that was also unintentional, right? just checking before I re-add

Yes, it was all unintentional.